README.md

```diff
@@ -21,8 +21,8 @@ import Fastify from 'fastify'
 const fastify = Fastify()
 await fastify.register(import('@fastify/throttle'), {
   bytesPerSecond: 1024 * 1024 // 1MB/s
```

Comment:
Shouldn't there be a comma on this line as well?

Reply:
You're right—I missed a spot. I just fixed it.